test/unit/output/ferc1_test.py

```diff
@@ -37,7 +36,104 @@
 logger = logging.getLogger(__name__)
 
 
-class TestForestSetup(unittest.TestCase):
+@pytest.fixture
+def forest_setup():
```

Comment:
Pulling out just the *nodes* as a fixture doesn't help much, as you see you still have to unpack and add edges etc. The actual setup that has to get done for each test is "make a forest, with nodes, edges, and tags" - and each test has a slight variation on what those are. To me that means we actually want a helper function in this module that makes creating these forests a little less verbose, rather than a few fixtures.

But that involves a fair amount of restructuring of the tests, etc., and this PR has already gone on for a long time. I would just return to the xunit-style setup using the `setup_method` flow you use below.